user-registration-app/src/app/views/user-registration-view/user-registration-view.component.ts

```diff
@@ -121,7 +121,7 @@ export class UserRegistrationViewComponent implements OnInit {
 
   showClassName(class_name: string): boolean {
     const hiddenClassNames = ['Business Challenge', 'CMPEN 362', 'STAT 463', 'SCM 421\n','EE 300W',
```

Comment:
Is there a reason SCM 421 has a "\n" in it?

Reply:
so in the db it accidentally has a newline and if we change the name, the SQL match ups get messed up, so we have to escape it here